index.js

```diff
@@ -84,19 +84,29 @@ export class UpdateAPK {
     // You might check {totalSpace, freeSpace} = await RNFS.getFSInfo() to make sure there is room
     const downloadDestPath = `${RNFS.CachesDirectoryPath}/NewApp.apk`;
 
-    const ret = RNFS.downloadFile({
-      fromUrl: remote.apkUrl,
-      toFile: downloadDestPath,
-      begin,
-      progress,
-      background: true,
-      progressDivider
-    });
+    let options = this.options.apkOptions ? this.options.apkOptions : {};
+
+    const ret = RNFS.downloadFile(
+      Object.assign(
```

Comment:
I believe since react-native bundles it's own javascript VM we are allowed to use ES6 so the spread operator would be valid here, but in this case I don't think it would do anything for readability, so this should be fine